Tasks/FileTransform/filetransform.ts

```diff
@@ -0,0 +1,26 @@
+import tl = require('vsts-task-lib/task');
+import { Package } from 'webdeployment-common/packageUtility';
+var deployUtility = require('webdeployment-common/utility.js');
```

Comment:
use let.

Reply:
Should we use var for global variables?  am planning to use let within the function.